list/list.go

```diff
@@ -1048,17 +1060,24 @@ func (m Model) statusView() string {
 	totalItems := len(m.items)
 	visibleItems := len(m.VisibleItems())
 
-	plural := ""
-	if visibleItems != 1 {
-		plural = "s"
+	var title string
+	if m.statusBarTitle != "" {
+		title = fmt.Sprintf("%d %s", visibleItems, m.statusBarTitle)
+	} else {
+		plural := ""
+		if visibleItems != 1 {
+			plural = "s"
+		}
+
+		title = fmt.Sprintf("%d item%s", visibleItems, plural)
```

Comment:
I think this `if m.statusBarTitle != ""` statement can go away if we set the default value of `m.statusBarTitle` to `item` in the `New` constructor and then allow people to change it with `SetStatusBarTitle`.

Like we do with other defaults, here:
https://github.com/charmbracelet/bubbles/blob/a62c69a3a309a98fa35c97be01b8959113a522b9/list/list.go#L200-L221

Reply:
I'd suggest setting `statusBarTitle:     "item"` in the constructor and then simply use that value, whether it is the default or user customized.

```suggestion
	plural := ""
	if visibleItems != 1 {
		plural = "s"
	}

	title = fmt.Sprintf("%d %s%s", visibleItems, m.statusBarTitle, plural)
```

Also, I wonder if we can change the `statusBarTitle` name to something like `itemName` to show what is being filtered so that people use the singular form so we can handle the pluralization.

Reply:
Adding this to `New` constructor, we cannot handle plurals anymore. What do you think about to change the default to `item(s)` or something like that?

Reply:
Maybe we should have a `itemNameSingular` and `itemNamePlural` that should be set? I.e. if someone is listing people they should set `itemNameSingular = "person"` and `item = "people"` and then we can set the defaults to `item` and `items`? That way we can also handle pluralization for the custom settings as well by performing the logic when `visibleItems > 1`.

What do you think?
I'm also not opposed to doing `item(s)` as this keeps the changing less when filtering and then simply having `statusBarTitle` but I might need to ask the team about this.

Reply:
> Maybe we should have a itemNameSingular and itemNamePlural that should be set? I.e. if someone is listing people they should set itemNameSingular = "person" and item = "people" and then we can set the defaults to item and items? That way we can also handle pluralization for the custom settings as well by performing the logic when visibleItems > 1.

I like this approach.